src/app/BlockHelpers/blockHelpers.js

```diff
@@ -0,0 +1,10 @@
+// blockHelpers.js
+// Contain various helper functions for managing props as well as findinging specific
+// elements and blopcks within them
+
+// filterForBlockType
```

Comment:
could we remove this line?

Reply:
if you dont like comments sure.

Reply:
I think the comment is fine, just not sure we need the `filterForBlockType` line 😄